templates/commands/render/templatesource/localsource.go

```diff
@@ -42,18 +45,26 @@ func (l *localSourceParser) sourceParse(ctx context.Context, src, protocol strin
 	// This sourceParser should run after the sourceParser that recognizes remote
 	// git repos, so this code won't run if the source looks like a git repo.
 
-	_, err := os.Stat(src)
-	if err != nil {
+	// If the filepath was not absolute, convert it to be relative to the cwd.
+	originalSrc := src
+	if !filepath.IsAbs(src) {
+		src = filepath.Join(cwd, src)
+		logger.DebugContext(ctx, "source path is relative, converting to absolute path",
+			"originalSrc", originalSrc,
+			"src", src)
+	}
+
+	if _, err := os.Stat(src); err != nil {
 		var pathErrPtr *fs.PathError
 
 		if errors.Is(err, fs.ErrNotExist) || errors.Is(err, fs.ErrInvalid) || errors.As(err, &pathErrPtr) {
-			logger.DebugContext(ctx, "won't treat src as a local path because that path doesn't exist", "src", src)
+			logger.DebugContext(ctx, "ignoring src as a local path because the path does not exist", "src", src)
```

Comment:
The word "ignoring" seems likely to be misinterpreted by the user. This particular function is indeed ignoring it, but other code may not ignore it. It's probably asking too much of the user to understand that distinction.

Reply:
Reverted to "will not" (contractions are bad for non-native speakers).